styles.css

```diff
@@ -3,78 +3,103 @@
     padding: 0;
     box-sizing: border-box;
 }
-.app {
-    width: 100%;
-    height: 100%;
-    background: url(images/background.png);
-    background-repeat: no-repeat;
-    background-size: cover;
-}
+
 .vector {
     list-style: none;
     display: flex;
     gap: 10px;
 }
-.nav-bar-list {
-    display: flex;
-    gap: 32px;
-   align-items: center;
-}
-.nav-bar-list-right {
+.feature-section {
     display: flex;
-    gap: 12px;
-   align-items: center;
+    flex-direction: column;
+    padding: 64px 0 0;
```

Comment:
make it padding: 65px 0 bcoz I think padding is the same for both the top and bottom.